### ISSUE TITLE
To get floating window popup in app drawer and ability to add home screen shortcut to open yamf^2 app tray to launch app in floating window directly without going in quick panel or opening app

### ISSUE THREAD

**Issue by Ankit2502** [kaii-lb/YAMFsquared#2]
### Description

would like to ask for 2 feature.. (I am practically begging for these feature to all developer of floating window but everyone seems to give up on there project, hope you would add these feature)

Get floating window hook in app drawer too.
You can get clear idea what I want to say from below video
https://github.com/duzhaokun123/YAMF/assets/89592455/6289ede7-a358-4bd8-aa1d-870bdac74c93

Ability to open app list directly from any shortcut which can be added on home screen so that tablets user can add that shortcut in taskbar or anywhere and open any app in floating window by simply opening the app list of yamf^2 app

### Describe the solution you'd like

Ability to open the any application in the floating window from the app drawer directly even when any application is open at first
From this video you can get clear idea what I exactly want to say
https://github.com/duzhaokun123/YAMF/assets/89592455/6289ede7-a358-4bd8-aa1d-870bdac74c93

Basically a shortcut of open app list of yamf app so that I can add that shortcut in the taskbar so whenever if I want to open any application in the floating window, I can just open that app list and open the application in the floating window.
For context, I'm talking about this app list.
Screenshot_20240511-213506_YAMF^2

Instead of going in application clicking on three dot menu and opening app list from there, it would be very good if we can create any shortcut or activity shortcut of that app list tray on home screen and add that shortcut in taskbar to open apps in floating window directly.

+ Also if you can lower the minimum sdk to 30 or 31 that would be really nyc for many older android version users..

**Comment by kaii-lb** [comment]
for first issue, if youre using a launcher3 like launcher then you can longpress on the app icon and launch it in freeform that way.
you need to enable it in yamfsquared settings tho

second one i'll think about it, but for now theres a quicksettings tile that does what you want
